Hazel/src/Platform/Windows/WindowsWindow.cpp

```diff
@@ -48,6 +48,13 @@ namespace Hazel {
 			s_GLFWInitialized = true;
 		}
 
+		#ifdef HZ_PLATFORM_MACOS
```

Comment:
Might want to checkout PR #137 its doing something similar, with a comment suggesting this approach instead.